guides/getting-started/set-up/indy-vdr.md

```diff
@@ -0,0 +1,174 @@
+# Indy VDR
+
+[Hyperledger Indy VDR](https://github.com/hyperledger/indy-vdr), Verifiable Data Registry, can be used to connect to one or more Indy Node ledger pools given sets of genesis transactions. Methods are provided to construct ledger requests and send them to the validators, collecting the results and ensuring that there is a consensus between the nodes. In the context of Aries Framework JavaScript, we mainly leverage it to register, and resolve, schemas, credential definitions and dids.
+
+:::caution
+
+Support for Indy VDR in Aries Framework JavaScript is currently experimental. We recommend new projects to use Indy VDR from the start, and also to migrate existing projects to Indy VDR, from the indy-sdk. However, projects may experience some issues. If you encounter any issues, please [open an issue](https://github.com/hyperledger/aries-framework-javascript/issues/new).
+
+Currently, there are few limitations to using Indy VDR.
+
+- React Native projects leveraging [Expo](https://expo.dev) are not supported at the moment. Support for Expo will be added soon.
+- React Native Android support starts from 11 right now. Work is being done on support for lower versions.
+- When running in Node.JS, only Node.JS 18 is suported for now. See [Supported Node.JS versions for Indy VDR](#supported-nodejs-versions-for-indy-vdr)
+
+:::
+
+### Installing Indy VDR
+
+When using Aries Framework JavaScript with Indy VDR, there are a few extra dependencies that need to be installed. We need to install the `@hyperledger/indy-vdr`, which contains all the functionality to register objects on an Hyperledger Indy VDR. Secondly, we need to add native bindings for the specific platform `@hyperledger/indy-vdr-<platform>`. Currently there are bindings for Node.JS, as `@hyperledger/indy-vdr-nodejs`, and React Native as `@hyperlegder/indy-vdr-react-native`.
+
+<!--tabs-->
+
+# Node.JS
+
+```console
+yarn add @aries-framework/indy-vdr @hyperledger/indy-vdr-nodejs
+```
+
+# React Native
+
+```console
+yarn add @aries-framework/indy-vdr @hyperledger/indy-vdr-react-native
+```
+
+<!--/tabs-->
+
+### Configuration
+
+As you can see below, the Indy VDR module takes the native bindings and a list of networks. This list of networks will be used to resolve and register objects on.
+
+```typescript title="example"
+import { indyVdr } from '@hyperledger/indy-vdr-nodejs'
+import { IndyVdrModule } from '@aries-framework/indy-vdr'
+
+indyVdr: new IndyVdrModule({
+  indyVdr,
+  networks: [
+    {
+      indyNamespace: 'bcovrin:test',
+      isProduction: false,
+      genesisPath: './path/to/genesis.txn',
+      connectOnStartup: true,
+    },
+  ],
+})
+```
+
+#### indyVdr
+
+**Type**: `IndyVdr`
+
+the `indyVdr` key takes a class that implements all the native bindings for Indy VDR. This can be extracted from the `@hyperledger/indy-vdr-nodejs` package or the `@hyperledger/indy-vdr-react-native` package.
```

Comment:
```suggestion
the `indyVdr` key takes a class that implements all the native bindings for Indy VDR. This can be imported from the `@hyperledger/indy-vdr-nodejs` package or the `@hyperledger/indy-vdr-react-native` package.
```